components/x-podcast-launchers/src/PodcastLaunchers.jsx

```diff
@@ -96,7 +96,7 @@ class PodcastLaunchers extends Component {
 				</div>
 
 				<h2 className={styles['heading']}>Can't see your podcast app?</h2>
```

Comment:
Sorry I should've noticed this before... can we make this a curly apostrophe? (`’`)

Reply:
ohh, I will change it to a curly apostrophe! Thank you!

Reply:
Thanks, sorry it is so minor 🙈